README.md

```diff
@@ -66,7 +66,7 @@ People use GitHub to build some of the most advanced technologies in the world.
   Encourage users to open new tabs for steps!
 -->
 
-<details id=1>
+<details id=1 open>
 <summary><h2>Step 1: Create a branch</h2></summary>
 
 _Welcome to "Introduction to GitHub"! :wave:_
```

Comment:
This is me suggesting a change.  I couldn't find the button that was in the tutorial.